### ISSUE TITLE
Unable to communicate with HCI device

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/Adafruit_CircuitPython_AirLift#4]
I have tried the airlift_simpletest on several devices and all are failing as below:
I have tried the Pyportal Titano, feather M4+Airlift Featherwing (with GPIO0,TX,RX jumpers soldered) and Feather RP2040 woth airlift featherwing.

Wifi works fine on all.
the Titano has NINA 1.7.3  the Featherwing has 1.7.4
I was able to update the Feather wing so that should have verified that all the jumpers were connected.

Any idea what is going wrong here?

```

Adafruit CircuitPython 7.0.0-alpha.4-64-ge7c5165ff on 2021-07-14; Adafruit PyPortal Titano with samd51j20
>>> 
>>> 
>>> import airlift_simpletest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "airlift_simpletest.py", line 11, in <module>
  File "adafruit_airlift/esp32.py", line 185, in start_bluetooth
_bleio.BluetoothError: Timeout waiting for HCI response
>>> 
```

**Comment by RLovelett** [comment]
I am using the [Adafruit AirLift breakout](https://www.adafruit.com/product/4201) which has `Firmware vers. bytearray(b'1.7.4\x00')` with a Raspberry Pi Pico.

I am trying to follow along with [Quickstart - Raspberry Pi RP2040 with BLE and CircuitPython](https://learn.adafruit.com/quickstart-raspberry-pi-rp2040-with-ble-and-circuitpython).

When I call `esp32.start_bluetooth()` it fails with:

```txt
Adafruit CircuitPython 7.0.0 on 2021-09-20; Raspberry Pi Pico with rp2040
>>> 
>>> 
soft reboot

Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Traceback (most recent call last):
  File "code.py", line 20, in <module>
  File "adafruit_airlift/esp32.py", line 185, in start_bluetooth
_bleio.BluetoothError: Timeout waiting for HCI response

Code done running.
```

The CircuitPython in my `code.py` is:

```python
import board
from adafruit_ble import BLERadio
from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
from adafruit_ble.services.nordic import UARTService
from adafruit_airlift.esp32 import ESP32

esp32 = ESP32(
   reset=board.GP16,
   gpio0=board.GP9,
   busy=board.GP14,
   chip_select=board.GP13,
   tx=board.GP0,
   rx=board.GP1,
)

adapter = esp32.start_bluetooth()
```

The error seems it is the same as yours @jerryneedell did you ever get a solution to yours?

**Comment by AP4130** [comment]
Just wanted to be sure this issue was crosslinked from the main CP page:
https://github.com/adafruit/circuitpython/issues/5462#issuecomment-960916269

**Comment by FoamyGuy** [comment]
The core issue this was linked to had a fix that went live in version `7.1.0-beta.2`. 

I think this can be closed now. If anyone is still having issues feel free to make a new issue and supply the versions of circuitpython and libraries in use.
